src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/MethodCallExpression.cs

```diff
@@ -1162,6 +1163,7 @@ public static MethodCallExpression Call(Expression instance, string methodName,
         /// <exception cref="ArgumentNullException">
         /// <paramref name="type"/> or <paramref name="methodName"/> is null.</exception>
         /// <exception cref="InvalidOperationException">No method whose name is <paramref name="methodName"/>, whose type parameters match <paramref name="typeArguments"/>, and whose parameter types match <paramref name="arguments"/> is found in <paramref name="type"/> or its base types.-or-More than one method whose name is <paramref name="methodName"/>, whose type parameters match <paramref name="typeArguments"/>, and whose parameter types match <paramref name="arguments"/> is found in <paramref name="type"/> or its base types.</exception>
+        [RequiresUnreferencedCode(GenericMethodRequiresUnreferencedCode)]
         public static MethodCallExpression Call(
```

Comment:
There's an intrinsic for Call in linker, so the annotations are not actually used by the linker (but they should remain for sure). Unfortunately there's a bug which will make all the generic method cases not warn: https://github.com/mono/linker/issues/1819

Needs to be fixed,